website/docs/releases/2025/v2025.6.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Release 2025.6
+slug: "/releases/2025.6"
+---
+
+## Highlights
+
+- **mTLS** :ak-enterprise The Mutual TLS stage enables authentik to use client certificates to enroll and authenticate users. These certificates can be local to the device or available via PIV Smart Cards, Yubikeys, etc. For environments where certificates are already rolled out, this can make authentication a lot more seamless.
+
+- **Email verification compatibility with link scanners**: Improved compatibility with certain email providers or scanning software which would previously invalidate links sent by authentik.
+
+- **LDAP source sync forward deletions**: This option will remove users created via LDAP sources when they are removed in the LDAP source.
+
+## Breaking changes
+
+- **Helm chart dependencies upgrades**:
+
+    - The PostgreSQL chart has been updated to version 16.7.4. The PostgreSQL image is no longer pinned in authentik's default values and has been upgraded from version 15 to 17. Follow [our PostgreSQL upgrade instructions](https://docs.goauthentik.io/docs/troubleshooting/postgres/upgrade_kubernetes) to update to the latest PostgreSQL version.
+    - The Redis chart has been updated to version 21.1.6. There are no breaking changes and Redis has been upgraded from version 7 to 8.
+
+- **Deprecated and frozen `:latest` container image tag after 2025.2**
+
+    Using the `:latest` tag with container images is not recommended as it can lead to unintentional updates and potentially broken setups.
+
+    The tag will not be removed, however it will also not be updated past 2025.2.
+
+    We strongly recommended the use of a specific version tag for authentik instances' container images, such as `:2025.4`.
+
+- **CSS**: We’ve made some improvements to our theming system. If your authentik instance uses custom CSS, you may need to review flow and user interfaces for any visual changes.
+
+## New features and improvements
+
+- **mTLS**: The Mutual TLS stage enables authentik to use client certificates to enroll and authenticate users. These certificates can be local to the device or available via PIV Smart Cards, Yubikeys, etc.
```

Comment:
```suggestion
- **mTLS**: The Mutual TLS stage enables authentik to use client certificates to enroll and authenticate users. These certificates can be local to the device or available via PIV Smart Cards, Yubikeys, etc. For environments where certificates are already rolled out, this can make authentication a lot more seamless.
```